lib/KListWithOverflow.vue

```diff
@@ -254,17 +264,17 @@
 
 
 <style scoped>
-
   .list-wrapper {
     display: flex;
-    justify-content: space-between;
+    justify-content: space-between; 
```

Comment:
Lets add a `alignItems: center` instruction here, so the default behaviour is that the items are centered